wemake_python_styleguide/logic/nodes.py

```diff
@@ -26,3 +27,14 @@ def get_parent(node: ast.AST) -> Optional[ast.AST]:
 def get_context(node: ast.AST) -> Optional[ContextNodes]:
     """Returns the context or ``None`` if node has no context."""
     return getattr(node, 'wps_context', None)
+
+
+def evaluate_node(node: ast.AST) -> Optional[Union[int, float, str, bytes]]:
+    """Returns the value of a node or its evaluation."""
+    if isinstance(node, ast.Name):
+        return None
+    try:
+        signed_node = literal_eval_with_names(node)
```

Comment:
This can return `ast.Name`. I guess - we don't expect that, do we?

Reply:
No, `literal_eval_with_names` does not return `ast.Name`. If anything, it could return the id of the variable representing `ast.Name`, only if `node` is an instance of it, but I take care of that right before calling this function.

Reply:
Oh, yes! Thanks you for reminding 🙂 
https://github.com/wemake-services/wemake-python-styleguide/blob/master/wemake_python_styleguide/logic/safe_eval.py#L16

Reply:
May I then resolve this conversation?